silkworm/db/kv/api/local_timestamp.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+   Copyright 2024 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#include "local_timestamp.hpp"
+
+#include <silkworm/infra/common/ensure.hpp>
+
+namespace silkworm::db::kv::api {
+
+datastore::TimestampRange ts_range_from_kv(Timestamp from_ts, Timestamp to_ts, bool reverse) {
```

Comment:
```suggestion
datastore::TimestampRange ts_range_from_kv(TimestampRange ts_range, bool reverse) {
```
TimestampRange is defined in endpoint/common.hpp